commands/tone.ts

```diff
@@ -51,13 +51,16 @@ export const translate = async (
   // Determine what to analyze.
   const split: string[] = message.content.split(" ");
   let text = "";
-
   // check if the message is after the !tone command
   if (split.length >= 2) {
     text = split.slice(1).join(" ");
   } else {
     text = messageToAnalyze.content.trim();
   }
+  let cutText: string[] = text.split(" ");
+  if (cutText.length >= 10) {
+    text = cutText.slice(0, 10).join(" ") + " ...";
+  }
```

Comment:
cuts the text after ten word.
ex input text: hello all, I have written this part to cut the text after ten word
output text: hello all, I have written this part to cut the ...
if the text is less then 10 word it will return the whole text.